packages/plugin-devtools/src/components/InspectorComponent.ts

```diff
@@ -88,7 +88,15 @@ export class InspectorComponent extends OwlComponent<InspectorComponentProps> {
         this.selectedNode = this.getNode(this.state.selectedID);
     }
     getNode(id: number): VNode {
-        return this.domEngine.root.descendants(node => node.id === id)[0] || this.domEngine.root;
+        const stack: VNode[] = [this.domEngine.root];
+        for (const node of stack) {
+            if (node.id === id) {
+                return node;
+            } else {
+                stack.push(...node.childVNodes);
+            }
+        }
+        return this.domEngine.root;
```

Comment:
Je trouve ça très confus comme construction : tu fais un `for ... of` sur un array de 1 item mais tu ajoutes des items au fur et à mesure. Je suggère quelque chose du genre :
```ts
const root = this.domEngine.root;
return [root, ...root.descendants()].find(node => node.id === id) || root;
```

Reply:
...merci au perf... de ne pas faire ce genre de chose. Les stacks évide les max call de fonction et sont plus performantes.

Typiquement, ce genre de chose: `return [root, ...root.descendants()].find(node => node.id === id) || root;` sont des trucs que je retire pour que les perf soient bonnes. Aller chercher tous les enfants avec 3 indirection de fonction et puis faire le filtre...